nepalingo-web/index.html

```diff
@@ -4,7 +4,9 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>Nepalingo</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
```

Comment:
A better way to do this is by using an npm package. You can follow the directions here:

https://docs.fontawesome.com/web/use-with/react/

If this is too much for this MR, we can add it as a followup ticket

Reply:
Let's make a followup ticket

Reply:
created a followup ticket here: https://github.com/nepalcodes/nepalingo/issues/59